README.md

```diff
@@ -122,6 +122,17 @@ drink = Drink.create(name: "Paloma", user: bryan)
 drink.user == bryan #=> true
 ```
 
+Associations are also exposed as "bang methods" which raise a
+`DocumentNotFoundError` if the record pointed to by the foreign key is no longer
```

Comment:
IMO we should fully-qualify the exception (`Minidoc::DocumentNotFoundError`) here and below